client/validators/profile.ts

```diff
@@ -40,7 +48,7 @@ export const formProfile = ({field, value, profile, errors, messages, diff}) =>
             errors[field] = gettext('Too long');
             messages.push(gettext('{{ name }} is too long', {name: fieldLabel}));
         }
-    } else if (schema.required && !schema.multilingual && isValueEmpty) {
+    } else if (schema.required && !schema.multilingual && valueIsValid) {
```

Comment:
actually `valueIsValid` isn't right, because if it is valid, why do you display an error that field is required? It should either be `!valueIsValid` or `!hasValidValue`